tests/wpt/metadata/css/CSS2/selectors/first-letter-punctuation-341.xht.ini

```diff
@@ -1,3 +1,3 @@
 [first-letter-punctuation-341.xht]
   type: reftest
-  expected: FAIL
+  expected: TIMEOUT
```

Comment:
Hmm, this is third time 🤔 I think it might not need to be updated as TIMEOUT?

```
{
    "status": "FAIL", 
    "group": "default", 
    "message": "/css/CSS2/selectors/first-letter-punctuation-341.xht 07bd1ae84c8390ac3e1d662deeaeef2c5a7f392b\n/css/CSS2/selectors/first-letter-punctuation-341-ref.xht 732b170530b51aab0fe1c83a60a8b82dbf7a22c8", 
    "stack": null, 
    "subtest": null, 
    "test": "/css/CSS2/selectors/first-letter-punctuation-341.xht", 
    "line": 83028, 
    "action": "test_result", 
    "expected": "TIMEOUT"
}
```